web-client/src/presenter/actions/TrialSession/getBulkSpecialTrialSessionCopyNotesAction.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * get bulk special trial session copy notes
+ * @param {object} providers the providers object
+ * @param {object} providers.applicationContext the application context
```

Comment:
JSDOC + add types for props.trialSessions coming in. 

An example of typed props in actions is web-client/src/presenter/actions/confirmSignUpAction.ts